docs/docs/spec/metrics/metrics-prom.md

```diff
@@ -7,20 +7,38 @@ id: metrics-prom-spec
 
 The table below defines metrics which may be captured by AVS Nodes which expose metrics to Prometheus. AVSs may expose additional metrics however these should not use the `eigen_` prefix. 
 
+## Economics metrics
+
+| Name | Metric Type | Definition | Labels |
+|---|---|---|---|
+| `eigen_fees_earned_total` | Counter | The amount of fees earned in `<unit>` of underlying `<token>` in the `<strategy>` contract. | `token`, `unit`, `strategy` |
```

Comment:
adapt according to notes I added above. For eg add that fees and slashing are optional and not present in the upcoming testnet launch.

Reply:
Done [d0e911](https://github.com/NethermindEth/el-node-spec/pull/58/commits/3d0e911b83f03ccb7740f16dddcf4ed43ee53e72)